### PR TITLE
update demos

### PR DESCRIPTION
**Pull request by MarcSabatella**
Reunion is re-entered into MuseScore 3 from scratch.  I kept it as an MSCZ because I used an image and stacking order tricks to get a "broken tie" effect at the end.

I retired "All Dudes" and "Triumph" and added a new leadsheet "Unclaimed Gift" and a fugue for string quartet.  These are MSCX because, why not?

There's room for plenty more...

### CODE REVIEW

**Comment by shoogle** [comment]
Indeed it is good to use MSCX as text files are stored more efficiently by Git. You can do the same for Reunion by unzipping the MSCX and putting the contents inside a "Reunion" directory, then add a small amount of code to `demos/CMakeLists.txt` to have CMake zip it at build time. I will update this comment with the necessary code once I have checked it works.

File: demos/CMakeLists.txt [hunk 1]
```diff
@@ -18,9 +18,9 @@
 #=============================================================================
 
 install(FILES
-      All_Dudes.mscz
+      Fugue_1.mscx
       Reunion.mscz
-      Triumph.mscz
+      Unclaimed_Gift.mscx
```
Comment (Jojo-Schmitz) [inline demos/CMakeLists.txt]: But you're leaving All_Dudes.mscz and Triumph.mscz in the repository?

**Comment by Jojo-Schmitz** [comment]
We may need something similar to convert all the templates from mscx to mscz on `make install`, maybe using a `.json` file and feeding that into MuseScore. for the templates such a .jscon file exists already, just no mechanism to use it

File: demos/CMakeLists.txt (hunk 1 above)
Comment (shoogle) [reply demos/CMakeLists.txt]: If they are already in the repository then it is too late... unless they need to be changed of course, in which case they should be replaced with MSCX.

Comment (MarcSabatella) [reply demos/CMakeLists.txt]: I hate removing history anyhow.  There are other obsolete files there too.

**Comment by MarcSabatella** [comment]
I do like the idea of MSCX and converting during build, but FWIW I don't anticipate these files changing very often.  For now let's just get them in as is.

**Pull request merged**